.github/workflows/ci.yaml

```diff
@@ -29,7 +35,7 @@ jobs:
     # - run: sudo chmod 755 /usr/bin/google-chrome
     # - run: defaults write com.google.chrome HardwareAccelerationModeEnabled -integer 1
     # - run: export DISPLAY=:0
-    - run: npm run deploy
+    - run: npm run deploy --verbose
```

Comment:
I've added `--verbose` because this is a CI job.
1. If the CI is green, it's very likely nobody will look further at the verbose output.
2. If the CI is Red, then it saves time to already have verbose output instead of needing to clone and investigate for each of the jobs in the job matrix.

Currently I believe the CI only runs on 16.04.7 with nodejs 8 and nodejs10
nodejs 8 is EOL, nodejs10 is in maintenance only mode, and 16.04 has less than a year left to EOL, so it's very likely a contributor is running a different combination of linux + nodejs and is relaying on this CI job (if enabled on PRs see comment above) to ensure that a contribution hasn't broken anything.